src/components/accounts/two_factor/EnableTwoFactor.js

```diff
@@ -82,24 +82,19 @@ export function EnableTwoFactor(props) {
     const handleNext = async () => {
         if (!initiated && !loading && !has2fa && isValidInput()) {
             let response;
-            try {
-                Mixpanel.track("2FA Initialize two factor start")
-                response = await dispatch(initTwoFactor(accountId, method))
-            } catch(e) {
-                Mixpanel.track("2FA Initialize two factor fail", {error: e.message})
-            } finally {
-                if (response && response.confirmed) {
-                    Mixpanel.track("2FA Deploy multisig start")
-                    try {
-                        await handleDeployMultisig()
-                        Mixpanel.track("2FA Deploy multisig finish")
-                    } catch(e) {
-                        Mixpanel.track("2FA Deploy multisig fail", {error: e.message})
+            await Mixpanel.withTracking("2FA Initialize two factor",
+                async () => response = await dispatch(initTwoFactor(accountId, method)),
+                () => {},
+                async () => {
```

Comment:
@vgrichina Please have a look at this function since I want you to double check whether this one I also code in correct format and do not break the function.